### PR TITLE
Fixes #26268, #26269 - graphql: add FactName and FactValue queries

### PR DESCRIPTION
**Pull request by kamils-iRonin**
This patch adds FactName and FactValue queries to the GraphQL api.

### CODE REVIEW

File: app/graphql/types/base_object.rb [hunk 1]
```diff
@@ -4,9 +4,25 @@ class BaseObject < GraphQL::Types::Relay::BaseObject
...
+      end
+
+      def has_many(name, type, null: true)
+        field name, type.connection_type, null: null,
+          resolve: proc { |object| CollectionLoader.for(object.class, name).load(object) }
+      end
+
+      def belongs_to(name, type, null: true)
+        field name, type, null: null,
+          resolve: proc { |object| RecordLoader.for(type.model_class).load(object.send("#{name}_id")) }
```
Comment (timogoebel) [inline app/graphql/types/base_object.rb]: Nothing we have to solve here, is it safe to assume the association id field is always generated by the `_id` suffix?

Comment (kamils-iRonin) [reply app/graphql/types/base_object.rb]: I've found a better way. We can use `object.class.reflect_on_association(name).foreign_key`. WDYT?

Comment (timogoebel) [reply app/graphql/types/base_object.rb]: That's what I was trying to lead you to! ;-)

**Comment by kamils-iRonin** [comment]
rebased and applied changes after review

**Review by timogoebel — APPROVED** [review]

**Comment by timogoebel** [comment]
Thanks, @kamils-iRonin.

**Pull request merged**